packages/opentelemetry-api/src/metrics/Observation.ts

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ValueObserver } from './Metric';
+
+/**
+ * Interface for updating value of certain value observer
+ */
+export interface Observation {
+  valueObserver: ValueObserver;
```

Comment:
Should this be renamed `observer` in case we have other types of observers in the future?